unit-tests/src/test/java/org/eclipse/collections/impl/list/fixed/AbstractMemoryEfficientMutableListTestCase.java

```diff
@@ -227,8 +266,116 @@ public void aggregateByNonMutating()
     {
         Function<String, String> groupBy = Functions.getStringPassThru();
         Function2<Integer, String, Integer> sumAggregator = (aggregate, value) -> aggregate + Integer.parseInt(value);
-        MapIterable<String, Integer> actual = this.classUnderTest().aggregateBy(groupBy, () -> 0, sumAggregator);
-        MapIterable<String, Integer> expected = FastList.newList(this.classUnderTest()).aggregateBy(groupBy, () -> 0, sumAggregator);
+        MapIterable<String, Integer> actual =
+                this.classUnderTest().aggregateBy(groupBy, () -> 0, sumAggregator);
+        MapIterable<String, Integer> expected =
+                FastList.newList(this.classUnderTest()).aggregateBy(groupBy, () -> 0, sumAggregator);
```

Comment:
A few unrelated changes here?

Reply:
Yes.  A general rule I like to apply.  Don't make me scroll.